Dockerfile

```diff
@@ -8,17 +8,19 @@ RUN npm install
 COPY . .
 
 RUN npm run compile
-RUN npm test
-RUN npm run generateDocs
+# RUN npm test
```

Comment:
Reminder to get the commented parts back

Reply:
Amended the commit without this change.